sdk/tests/conformance/reading/00_test_list.txt

```diff
@@ -1,3 +1,3 @@
 read-pixels-pack-alignment.html
 read-pixels-test.html
-
+fbo-remains-unchanged-after-read-pixels.html
```

Comment:
I had added the min version for this test, please help to review it again, thank you.